lib/modules/filteReddit.js

```diff
@@ -746,6 +755,17 @@ module.go = function() {
 			toggleNsfwFilter(toggle, true);
 		}
 	);
+
+	if (isPageType('linklist')) {
+		if (module.options.showRemoveVisitedTab.value && process.env.BUILD_TARGET !== 'safari') {
+			CreateElement.tabMenuItem({
```

Comment:
> one would then have to scroll before initiating the option.

Maybe that "visibleAfterScroll" floating island should start below the header, rather than hidden, then pin to top when the header is out of view.